components/toggle/Toggle.js

```diff
@@ -0,0 +1,64 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import theme from './theme.css';
+import cx from 'classnames';
+import omit from 'lodash.omit';
+
+class Toggle extends PureComponent {
+  static propTypes = {
+    checked: PropTypes.bool,
+    disabled: PropTypes.bool,
+    name: PropTypes.string,
+    className: PropTypes.string,
+    onChange: PropTypes.func,
+    size: PropTypes.oneOf(['small', 'medium', 'large']),
+    color: PropTypes.oneOf(['neutral', 'teal', 'ruby', 'mint', 'violet', 'gold', 'aqua']),
+  };
+
+  static defaultProps = {
+    checked: false,
+    disabled: false,
+    size: 'medium',
+    color: 'neutral',
+  };
+
+  constructor() {
+    super(...arguments);
+    this.handleToggle = ::this.handleToggle;
+  }
+
+  handleToggle() {
+    if (!this.props.disabled && this.props.onChange) {
+      this.props.onChange(!this.props.checked, event);
+    }
+  }
+
+  render() {
+    const { checked, disabled, className, size, color, ...others } = this.props;
+    const rest = omit(others, ['onChange']);
+
+    const classNames = cx(theme['toggle'], className, theme[size], theme[color], {
```

Comment:
I think that **className** should be the last one in row to allow overriding

Reply:
Good catch 👍